src/overlays/actors/ovl_Oceff_Wipe6/z_oceff_wipe6.c

```diff
@@ -22,12 +21,80 @@ const ActorInit Oceff_Wipe6_InitVars = {
     (ActorFunc)OceffWipe6_Draw,
 };
 
-#endif
+/* Parameters for gOceffScroll */
+UNK_TYPE1 gOceff6ScrollParams[] = { 0xFF, 0x01, 0x20, 0x20, 0xFF, 0x04, 0x20, 0x20 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Oceff_Wipe6/OceffWipe6_Init.s")
+/* Animated Material Properties for Scrolling Texture */
+AnimatedMaterial gOceff6Scroll = { 0xFF, 0x0001, gOceff6ScrollParams };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Oceff_Wipe6/OceffWipe6_Destroy.s")
+#include "overlays/ovl_Oceff_Wipe6/ovl_Oceff_Wipe6.c"
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Oceff_Wipe6/OceffWipe6_Update.s")
+void OceffWipe6_Init(Actor* thisx, GlobalContext* globalCtx) {
+    OceffWipe6* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Oceff_Wipe6/OceffWipe6_Draw.s")
+    Actor_SetScale(&this->actor, 1.0f);
+    this->counter = 0;
+    this->actor.world.pos = GET_ACTIVE_CAM(globalCtx)->eye;
+}
+
+void OceffWipe6_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    func_80115D5C(&globalCtx->state);
+    globalCtx->msgCtx.unk_120B0 = 0;
+}
+
+void OceffWipe6_Update(Actor* thisx, GlobalContext* globalCtx) {
+    OceffWipe6* this = THIS;
+
+    this->actor.world.pos = GET_ACTIVE_CAM(globalCtx)->eye;
+    if (this->counter < 100) {
+        this->counter++;
+    } else {
+        Actor_MarkForDeath(&this->actor);
+    }
+}
+
+void OceffWipe6_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    OceffWipe6* this = THIS;
+    f32 z;
+    u8 alpha;
+    s32 i;
+    s32 counter;
+    Vec3f activeCamEye;
+    s32 pad;
+    Vec3f vec;
```

Comment:
Looks like this is the result from `Camera_GetSkyboxOffset` so perhaps `skyboxOffset` as a name?

Reply:
```suggestion
    Vec3f quakeOffset;
```
`quakeOffset` would be the proper name for this

Reply:
Acknowledged and modified accordingly.